hack/setup/README.md

```diff
@@ -0,0 +1,466 @@
+
+    az acr create --name <REGISTRY_NAME> --resource-group <RESOURCE_GROUP>
+</details>
+
+
+### After creation of an AKS Cluster:
+For your convenience, Fleet provides a script that can automate the process of creating a hub cluster. To use script,
+run the commands bellow:
+```sh
+# Replace the value of RESOURCE_GROUP with the name of the resource group your AKS cluster is in
+export RESOURCE_GROUP=<YOUR-RESOURCE-GROUP>
+# Replace the value of CLUSTER_NAME with the name of your AKS cluster you are using as a hub cluster
+export CLUSTER_NAME=<YOUR-CLUSTER-NAME>
+
+# Replace the value of HUB_CLUSTER_CONTEXT with the name of the kubeconfig context you use for
+# accessing your hub cluster.
+export HUB_CLUSTER_CONTEXT=<YOUR-HUB-CLUSTER-CONTEXT>
+# Replace the value of HUB_CLUSTER_ADDRESS with the address of your hub cluster API server.
+export HUB_CLUSTER_ADDRESS=<YOUR-HUB-CLUSTER-ADDRESS>
+
+# Replace the value of REGISTRY with name of your registry
+export REGISTRY=<ACR_NAME>
+# Replace the value of TAG with the tag name
+export TAG=<TAG>
+
+# Clone the Fleet GitHub repository, if necessary
+git clone https://github.com/Azure/fleet.git
+
+# Run the script.
+chmod +x fleet/hack/setup/createHubCluster.sh
+./fleet/hack/setup/createHubCluster.sh
+```
+It may take a few minutes for the script to finish running. Once it is completed, verify that the hub-agent has been installed:
+```
+kubectl get pods -n fleet-system
+```
+
+Alternatively, if you would like to find out the exact steps the script performs, or if you feel like fine-tuning some of the steps, 
+you may create the hub cluster with the instructions below:
+<details>
+<summary>Creating a hub cluster from an AKS cluster</summary>
+
+1. Go into fleet directory
+    ```
+    cd fleet
+    ```
+
+2. Build the hub agent image
+    ```
+    export OUTPUT_TYPE="${OUTPUT_TYPE:-type=docker}"
+    make docker-build-hub-agent
+    ```
+
+   Check if the image is built. Find your registry with the tag previously exported.
+   ```
+   docker images
+   ```
+3. Logging into registry
+    ```
+    az acr login -n $REGISTRY
+    ```
+4. Push the image to the registry
+    ```
+    docker push $REGISTRY.azurecr.io/hub-agent:$TAG
+   ```
+5. Attach acr to the cluster
+    ```   
+    az aks update -n $CLUSTER_NAME -g $RESOURCE_GROUP --attach-acr $REGISTRY
+    ```
+
+6. Install the hub agent helm chart onto cluster
+    ```
+    helm install hub-agent charts/hub-agent/ \
+    --set image.pullPolicy=Always \
+    --set image.repository=$REGISTRY.azurecr.io/hub-agent \
+    --set image.tag=$TAG \
+    --set logVerbosity=2 \
+    --set namespace=fleet-system \
+    --set enableWebhook=false \
+    --set webhookClientConnectionType=service \
+    --set enableV1Alpha1APIs=false \
+    --set enableV1Beta1APIs=true \
+    --set resources.limits.cpu=4 \
+    --set resources.limits.memory=4Gi \
+    --set concurrentClusterPlacementSyncs=10 \
+    --set ConcurrentRolloutSyncs=10 \

```

Comment:
what is this?

Reply:
That was one of the parameters we added at the beginning but then ended up taking off. Forgot to take it off of the wiki I created that I copied this over from.